lib/Table/TableColumn.ts

```diff
@@ -920,6 +920,6 @@ function toNumber(value: string): number | null {
   return null;
 }
 
-function nullFunction(rowIndex: number) {
+function nullFunction() {
```

Comment:
This function is returned from `scaledValueFunctionForType`, doesn't callers of that function expect the returned function to accept one parameter?

Reply:
They do but it doesn't look like typescript minds: [example](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAZgJhgXhgCgJQC4ZgK4A2BKAfDAN4BQMNMATgKZR51i6EGUC+lloksOAEYU6bOjQBDOgHMADDnwBbAEYM6GUuyKbkZKrXpMWbRN17DMaBBkpA) . Happy to change it to `(_rowIndex: number)` if you think that's clearer

Reply:
I had a second look, and wrote something that roughly equals "the type signature states the returned function should take a number and return a number, so I suspect removing the parameter will cause problems in some future upgrade of TypeScript".

As a side note, the return type of `nullFunction` is `null` rather than `number`, so the type is already wrong in the current code base. The function body is just `return null`, so TypeScript must be able to infer that return type as well.

I feel this behavior of a type checker is really strange, so I had a third look. The function `getScaledValueFunctionForType` was added in #3594 and had a caller in `TableMixin.ts`, but that caller no longer seems to be present. Could `getScaledValueFunctionForType` (and its helper functions) be removed instead?

Reply:
Nice find thanks I'll remove it